payment_term_type/models/payment_term.py

```diff
@@ -33,4 +29,4 @@ def _compute_payment_type(self):
     payment_type = fields.Selection(
         [('credit', _('Credit')),
          ('cash', _('Cash'))],
-        string="Payment Type", compute='_compute_payment_type')
+        compute='_compute_payment_type')
```

Comment:
@deivislaya why are you removing the string here?

Reply:
Because, the field's name matches with the string , I get a warning de pylint type

```python 
[attribute-string-redundant] The attribute string is redundant. String parameter equal to name of variable [python/pylint]
```

Reply:
OK. Good. I didn't know we had that lint rule :)

It's OK then.